### ISSUE TITLE
Please add support for accessing "Cash" transactions via debit card.

### ISSUE THREAD

**Issue by siddukonduru** [jmfernandes/robin_stocks#145]
To calculate accurate gains, needed deposits, withdrawals and debit card usage ( which is withdrawals but not ACH type). So could you please provide one wrapper API to read debit card transactions as well. Thank you very much for your work.

**Comment by gloera** [comment]
I started to look at this a little bit since I had the same issue.  It looks like the request URL is:
>> https://minerva.robinhood.com/history/transactions
 
I just started playing with robin_stocks a couple days ago, so I have to look how to integrate this into the project.  I'm thinking the best place is in _account.py_:
`def get_card_trans(info=None):`

@jmfernandes  What's the procedure to contribute to your awesome project?

**Comment by jmfernandes** [comment]
@gloera yes I agree that the best place is account.py. to contribute make a PR request with your changes. Be sure to also add the function to __init__.py and to make a doc string in the same format as all the other functions.

**Comment by gloera** [comment]
> @gloera yes I agree that the best place is account.py. to contribute make a PR request with your changes. Be sure to also add the function to **init**.py and to make a doc string in the same format as all the other functions.

@jmfernandes, thanks for the great repo and work.  I just sent out the request.  Let me know if there's anything I missed.  Wasn't sure if I had to do something special to update the function documentation or if it's autogenerated.

**Comment by siddukonduru** [comment]
Thanks @gloera @jmfernandes for adding this feature. I tested this and found one small issue with the logic for "money_Invested".
money_invested = deposits - (withdrawals - debits)
shouldn't it be,
money_invested = deposits - (withdrawals + debits)

**Comment by sachinparyani** [comment]
Debit is like a deposit (direction-wise), so there is no problem in the logic

**Comment by siddukonduru** [comment]
No, i think the "debits" will hold the sum of transactions you did on debit card .. i mean purchases, so money is going out from Robinhood. So this money need to be removed from "money_invested" amount along with Withdrawal amount.

**Comment by gloera** [comment]
> Thanks @gloera @jmfernandes for adding this feature. I tested this and found one small issue with the logic for "money_Invested".
> money_invested = deposits - (withdrawals - debits)
> shouldn't it be,
> money_invested = deposits - (withdrawals + debits)

Yes you are correct, good catch.

**Comment by gloera** [comment]
@siddukonduru, I only have debits on my RH account, so I can't enumerate other **_'direction'_** types.  Do you have any additional transactions on your account?  I wasn't sure if a pending transaction shows up as **_'pending'_** or **_'debit'_**, and if you have a refund if it shows up as a **_'credit_'**.  If you have other card type transactions, please let me know what's returned in **_'direction'_** so I can update the example file to reflect those.

**Comment by siddukonduru** [comment]
@gloera ,mmm i too have all settled debit transactions will update you when i have pending transaction on my debit card, but no refund transaction yet.

**Comment by siddukonduru** [comment]
@gloera Please find the pending debit transaction,
{'account_id': 'ab49eeab-598e-4604-812f-***********', 'amount': {'currency_code': 'USD', 'currency_id': '1072fc76-1862-41ab-82c2-************', 'amount': '1.06'}, 'direction': 'debit', 'id': '068c49d3-3413-49f4-a179-58e314265180', 'initiated_at': '2020-08-06T20:14:34.251636-04:00', 'source_id': 'e03da740-c498-4985-ba8b-22e5cf93ceb2', 'source_type': 'pending_card_transaction', 'transaction_code': 'card_transaction', 'transaction_type': 'pending'}
